app/keepers/keepers.go

```diff
@@ -322,7 +326,7 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 		wasmDir,
 		wasmConfig,
 		availableCapabilities,
-		authtypes.NewModuleAddress(govtypes.ModuleName).String(),
+		govModuleAuthority,
```

Comment:
It asked for the address, I don't think you pass address in case it ain't devnet

Reply:
```
	govModuleAuthority := authtypes.NewModuleAddress(govtypes.ModuleName).String()
	if devnetGov != nil {
		govModuleAuthority = *devnetGov
	}
```